packages/showcase/src/app/components/tabular-grid/tabular-grid.component.html

```diff
@@ -1,12 +1,2703 @@
-<runtime-bundles [bundles]="['utils/typography']" />
+<runtime-bundles
+  [bundles]="[
+    'utils/typography',
+    'components/checkbox',
+    'components/tabular-grid'
+  ]"
+/>
 
 <header>
-  <h1 class="heading-1">Tabular Grid</h1>
+  <h1 class="heading-1">{{ metadata.title }}</h1>
 </header>
 
-<p>
-  This page is under construction and will be available soon. You can see the
-  <a class="hyperlink" href="/old/tabular-grid.html" target="_blank"
-    >old version here</a
-  >.
+<p class="text-body-regular-l">
+  {{ metadata.description }}
 </p>
+
+<ol class="filters elevation-1">
+  @for (
+    tabularGridEntry of this.tabularGrids().entries();
+    track tabularGridEntry[0]
+  ) {
+    @let tabularGridName = tabularGridEntry[0];
+
+    <li class="filters-checkbox">
+      <ch-checkbox
+        [caption]="tabularGridName"
+        class="checkbox"
+        [checkedValue]="'true'"
+        [value]="tabularGridEntry[1].toString()"
+        [unCheckedValue]="'false'"
+        (input)="updateRenderedTabularGrid(tabularGridName)($event)"
+      ></ch-checkbox>
+    </li>
+  }
+</ol>
+
+<!-- Tabular Grid -->
+
+@if (showSelectionNoneKeyboardFocus()) {
+  <code-snippet
+    [codeSnippet]="codeSnippets.selectionNoneKeyboardFocus"
+    [language]="'ts'"
+  >
+    <ch-tabular-grid
+      class="tabular-grid tabular-grid-cell-layout"
+      row-selection-mode="none"
+      keyboard-navigation-mode="focus"
+    >
+      <ch-tabular-grid-columnset class="tabular-grid-column-set">
+        <ch-tabular-grid-column
+          class="tabular-grid-column"
+          column-id="code"
+          column-name="Country Code"
+          settingable="false"
+          sortable="true"
```

Comment:
Use DOM properties instead of HTML attributes. You can use your own code examples to write the template and also test if the code example works.

```suggestion
      [rowSelectionMode]="'none'"
      [keyboardNavigationMode]="'focus'"
    >
      <ch-tabular-grid-columnset class="tabular-grid-column-set">
        <ch-tabular-grid-column
          class="tabular-grid-column"
          [columnId]="'code'"
          [columnName]="'Country Code'"
          [settingable]="false"
          [sortable]="true"
```